package/src/skia/core/Image.ts

```diff
@@ -7,4 +7,4 @@ import { useRawData } from "./Data";
  * Returns a Skia Image object
  * */
 export const useImage = (source: DataSource, onError?: (err: Error) => void) =>
-  useRawData(source, Skia.MakeImageFromEncoded, onError);
+  useRawData(source, Skia.Image.MakeImageFromEncoded, onError);
```

Comment:
this was available via ` MakeImageFromEncoded: SkiaApi.Image.MakeImageFromEncoded,` but not part of the `SkiaApi` type